server/world/world.go

```diff
@@ -186,6 +187,24 @@ func (w *World) Biome(pos cube.Pos) Biome {
 	return b
 }
 
+// BlocksAround returns all block positions around the AABB passed.
+func (w *World) BlocksAround(aabb physics.AABB) []cube.Pos {
```

Comment:
It is used in movement computer along with travel computers for detecting a colliding portal so I thought it would make sense, although if you feel otherwise I can unexport this and move it somewhere else.

Reply:
Removed.